src/components/footer.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./footerStyles.scss";
```

Comment:
`CSS modules`. It should be in `components/footer` and the `CSS Module` should be called `footer.module.scss` inside the `footer` directory.